v2/pkg/output/output.go

```diff
@@ -320,3 +321,22 @@ func (w *StandardWriter) WriteStoreDebugData(host, templateID, eventType string,
 	}
 
 }
+
+type ListScanOutput struct {
```

Comment:
let's keep this structure part of `v2/internal/cloud` package since we don't want cloud types combining with the rest of nuclei code particularly the public API